src/config/config.ts

```diff
@@ -604,6 +604,13 @@ export class Config implements IConfig {
     await this.runHook('prerun', {Command: command, argv})
 
     const result = (await command.run(argv, this)) as T
+    // If plugins:uninstall was run, we need to remove all the uninstalled plugins
+    // from this.plugins so that the postrun hook doesn't attempt to run any
+    // hooks that might have existed in the uninstalled plugins.
+    if (c.id === 'plugins:uninstall') {
+      for (const arg of argv) this.plugins.delete(arg)
```

Comment:
`plugins uninstall` only accepts 1 arg but I'm guessing `argv` could also include `--verbose`. `this.plugins` is a map and `delete` doesn't throw if an element doesn't exist so this seems safe.

Reply:
Yes but I think `plugins uninstall` is supposed to take multiple arguments: https://github.com/oclif/plugin-plugins/blob/main/src/commands/plugins/uninstall.ts#L16

I'm assuming at some point `variableArgs = true` was changed to `strict = false` but the command was never updated. Iterating over the args now future proofs us from that being fixed eventually